CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Upgrade the Windows build to use net6 now that net5 has reached end of life.
 - Set console output encoding to `utf-8` explicitly.
+- Upgrade MSAL to 4.47.2.
```

Comment:
I think you have got the older version of master. Current changelog has 0.6.0 release in it, so this will go in Unreleased section above 0.6.0